src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -263,6 +268,19 @@ export class SkyColorpickerInputDirective
     // Validation
   }
 
+  /**
+   * Implemented as part of ControlValueAccessor.
+   */
+   public setDisabledState(isDisabled: boolean) {
```

Comment:
Code style > your method will need a return type:
```
public setDisabledState(isDisabled: boolean): void {
```

Reply:
resolved in last commit